std/datetime.d

```diff
@@ -390,9 +390,14 @@ public:
             immutable result = FILETIMEToStdTime(&fileTime);
             static if(clockType == ClockType.second)
             {
-                // This should probably just use core.stdc.time.time, but dmc's
-                // time function seems to apply DST to time_t, which is
-                // incorrect and thus only works with dmc's other C functions.
+                // Ideally, this would use core.std.time.time, but the C runtime
```

Comment:
On the other hand, this is implementation. If I had to choose between comments I've seen elsewhere such as `@@@BUG@@@` with no explanation, and this detailed explanation, I'd rather see this.

In addition, this seems like an issue with various C runtimes, not D. Where do you file the issue? Note that the various runtimes are not violating C spec -- unix time is not guaranteed by C's `time_t` type. It has various functions to deal with this fact (`time_t` is supposed to be implementation defined). So it's not actually a bug.

I'll give you this, the explanation can probably be shorter :)